WayneSociety_Project/Routing.py

```diff
@@ -121,7 +121,3 @@ def Logout():
     logout_user()
     return redirect(url_for('Routing.Welcome'))
 
```

Comment:
Good changes, other than fixing crashing also made code easer to read by getting rid of unnecessary white space and combining lines 112-114 into one line